tests/telemetry_intake/update.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+
+# Define variables
+REPO_URL="https://github.com/DataDog/dd-go.git"
+SCRIPT_DIR=$(dirname "$(realpath "$0")")
+TARGET_DIR="$SCRIPT_DIR/static"
+SUBDIR="trace/apps/tracer-telemetry-intake/telemetry-payload/static/"
+BRANCH="bm1549/handle-missing-configs"  # This should ALWAYS be "prod" in master. Please revert changes before merging
```

Comment:
Blocked behind merge of https://github.com/DataDog/dd-go/pull/161301
```suggestion
BRANCH="prod"  # This should ALWAYS be "prod" in master. Please revert changes before merging
```